game.js

```diff
@@ -9,8 +9,11 @@ let simpleLevelPlan = `
 .....###############..
 ......................`;
 
-/**Now we define the level class. The descriptions must be written 
```

Comment:
You know I kind of wanna keep this messy comment thingy, because it's so nostalgic xD although I'm aware that's not a good reason to keep code.